Models/PaymentType.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+
+namespace BangazonAPI.Models
+{   
+    //    Structure in Database
+    //    CREATE TABLE PaymentType(
+    //    PaymentTypeId INTEGER NOT NULL PRIMARY KEY,
+    //    PaymentTypeName varchar(15) NOT NULL
+    //);
+    public class PaymentType
+    {
+        [Key]
+        public int PaymentTypeId { get; set; }
+        [StringLength(15, ErrorMessage = "Maxinum lenght is 15 characters"),Required]
+        public string PaymentTypeName { get; set; }
+    }
+}
```

Comment:
You are missing the CustomerId column

Reply:
@stevebrownlee , they decided as a team to have two separate tables for `PaymentType`and `CustomerPaymentType` . `CustomerPaymentType` will have a customer id and an account number (i.e. "Jordan's Visa"), but `PaymentType` will just have an id and a name. The `CustomerPaymentType` model is coming later- this is just `PaymentType`